.nvmrc

```diff
@@ -0,0 +1 @@
+>=16.14
```

Comment:
This will only be use if a user has nvm. Otherwise, the user's package manager will warn them if they're using an incorrect version, with a message similar to:

```sh
error @docusaurus/core@2.2.0: The engine "node" is incompatible with this module. Expected version ">=16.14". Got "16.13.0"
```